src/dataplane-protocol/Cargo.toml

```diff
@@ -25,7 +25,7 @@ derive_builder = { version = "0.9.0", optional =  true }
 
 # Fluvio dependencies
 fluvio-future = { version = "0.3.1" }
-fluvio-protocol = { path = "../protocol", version = "0.5.1", features = ["derive", "api"] }
+fluvio-protocol = { path = "../protocol", version = "0.6", features = ["derive", "api"] }
 flv-util = { version = "0.5.0" }
```

Comment:
bump up version

Reply:
This bumped version has not been published yet, so we should be good